src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/BLEUEvaluatorContext.cs

```diff
@@ -41,8 +41,8 @@ public sealed class BLEUEvaluatorContext : EvaluationContext
     /// <param name="references">
     /// The reference responses against which the response that is being evaluated is compared.
     /// </param>
-    public BLEUEvaluatorContext(params string[] references)
-        : this(references as IEnumerable<string>)
+    public BLEUEvaluatorContext(IEnumerable<string> references)
+        : this(references.ToArray())
```

Comment:
Based on a quick check, the same pattern of enumerating the supplied `IEnumerable` twice in the constructor also appears in the following `*Context` types. It would be nice to also fix those up such that the base constructor is called in the `params` overload...

```
IntentResolutionEvaluatorContext
RetrievalEvaluatorContext
TaskAdherenceEvaluatorContext
ToolCallAccuracyEvaluatorContext 
```